hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/TransactionInfo.java

```diff
@@ -162,7 +163,15 @@ public String toString() {
    */
   public static TransactionInfo readTransactionInfo(
       DBStoreHAManager metadataManager) throws IOException {
-    return metadataManager.getTransactionInfoTable().get(TRANSACTION_INFO_KEY);
+    return metadataManager.getTransactionInfoTable().getSkipCache(TRANSACTION_INFO_KEY);
```

Comment:
Why do we need getSkipCache() version for TransactionInfo ?

Reply:
We are always supposed to get the value from the disk, and never from the cache. There should be no cache present for transactionInfo table. I have added a fail safe check, to skip cache even if it exists.

Reply:
I think we agreed that there is no need to skip the cache because we don't update the cache directly for `TransactionInfoTable`. Once the transaction gets flushed, it updates the cache as well.

Reply:
Why even check the cache if the cache datastructure is not going to have it. SkipCache would directly go to rocksdb  and check it, in contrary to the get method making a roundtrip to cache and then going to rocksdb when not found in cache.